general-information.md

```diff
@@ -56,3 +84,111 @@ Users of beta features are strongly encouraged to share their experiences, learn
+| `electric_assist` | Provides power only alongside human propulsion         |
+| `electric`        | Contains throttle mode with a battery-powered motor    |
+| `combustion`      | Contains throttle mode with a gas engine-powered motor |
+
+A vehicle may have one or more values from the `propulsion`, depending on the number of modes of operation. For example, a scooter that can be powered by foot or by electric motor would have the `propulsion` represented by the array `['human', 'electric']`. A bicycle with pedal-assist would have the `propulsion` represented by the array `['human', 'electric_assist']` if it can also be operated as a traditional bicycle.
+
+## Vehicle States
+
+This table describes the list of vehicle conditions that may be used by regulators to assess the disposition of individual vehicles and fleets of vehicles.  Some of these states describe vehicles in the Public Right-of-Way (PROW), and others represent vehicles that are not.  One state (`unknown`) implies that PROW status is unknown.
+
+In a multi-jurisdiction environment, the status of a vehicle is per-jurisdiction.  For example, a vehicle may be in the `trip` status for a county that contains five cities, and also in the `trip` status for one of those cities, but `elsewhere` for the other four cities.  In such a condition, generally a Provider would send the device data to the over-arching jurisdiction (the county) and the vehicle state with respect to each city would be determined by the Agency managing the jurisdictions.
+
+| `vehicle_state` | In PROW? | Description |
+| --- | --- |
+| `removed`     | no | Examples include: at the Provider's warehouse, in a Provider's truck, or destroyed and in a landfill. |
+| `available`   | yes | Available for rental via the Provider's app. In PROW. |
+| `unavailable` | yes | Not available for rent.  Examples include: vehicle has low battery, or currently outside legal operating hours. |
+| `reserved`    | yes | Reserved via Provider's app, waiting to be picked up by a rider. |
+| `trip`        | yes | In posession of renter.  May or may not be in motion. |
+| `elsewhere`   | no | Outside of regulator's jurisdiction, and thus not subject to cap-counts or other regulations. Example: a vehicle that started a trip in L.A. has transitioned to Santa Monica.  |
+| `unknown`     | unknown | Provider has lost contact with the vehicle and its disposition is unknown.  Examples include: taken into a private residence, thrown in river. |
+
+## Vehicle Events
+
+This is the list of `vehicle_state` and `event_type` pairings that constitute the valid transitions of the vehicle state machine.
+
+Note that to handle out-of-order events, the validity of the prior-state is not enforced at the time of ingest via Provider or Agency.  Events received out-of-order may result in transient incorrect vehicle states.
+
+| Valid prior `vehicle_state` values | `vehicle_state` | `event_type` |  Description |
```

Comment:
Based on the conversation in the github issue, https://github.com/openmobilityfoundation/mobility-data-specification/issues/271#issuecomment-624501189, I was expecting to see an allowed transition from the `vehicle_state` `trip` to `unavailable` with the `event_type` `low_battery`. This would allow for vehicles that end a trip and are no longer available, ie the `{vehicle_state: trip, event_types: [...]) -> { vehicle_state: unavailable, event_types: [trip_end, low_battery] }` transition from the comment. A similar transition with the event_type `maintenance` could be used for vehicles where the trip ended in an accident and the vehicle is not made `available`.

Reply:
This raises an interesting point.  Because we can have multiple event_type values per event, then a single event could transition from any state to any other state.  This would make the notion of valid-prior-state meaningless, which is not exactly the intent.  Let's discuss at the next Provider meeting.

Reply:
I will add some clarifying verbiage.  Note to myself to do that. :)

Reply:
Thank you! This makes it much clearer to me.